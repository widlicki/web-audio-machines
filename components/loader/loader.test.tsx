import { render } from '@testing-library/react'
import Loader from '../loader/loader'

describe('Loader', () => {
  it('Should render Loader according to snapshot', () => {
    const { container } = render(<Loader />)
    expect(container).toMatchSnapshot()
  })
})
