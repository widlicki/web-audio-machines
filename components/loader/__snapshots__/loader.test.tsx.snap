// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Should render Loader according to snapshot 1`] = `
<div>
  <span
    class="MuiCircularProgress-root MuiCircularProgress-indeterminate MuiCircularProgress-colorPrimary css-18lrjg1-MuiCircularProgress-root"
    role="progressbar"
    style="width: 20px; height: 20px;"
  >
    <svg
      class="MuiCircularProgress-svg css-1idz92c-MuiCircularProgress-svg"
      viewBox="22 22 44 44"
    >
      <circle
        class="MuiCircularProgress-circle MuiCircularProgress-circleIndeterminate css-176wh8e-MuiCircularProgress-circle"
        cx="44"
        cy="44"
        fill="none"
        r="20.2"
        stroke-width="3.6"
      />
    </svg>
  </span>
</div>
`;
